app/views/posts/show.html.erb

```diff
@@ -26,4 +26,7 @@
       <%= render 'layouts/custom/comment_component', comment: comment %>
     <% end %>
   </div>
```

Comment:
## [OPTIONAL]

- [ ] Kindly note that your project contains some errors that need to be fixed.  The route to show the post is not working.

![image](https://user-images.githubusercontent.com/65068771/161164311-a9f16b29-9bd5-4c64-8003-7a8aad141edf.png)